src/index.js

```diff
@@ -104,6 +104,8 @@ export default class ToggleBlock {
     holder.setAttribute('foreignKey', foreignKey);
     holder.setAttribute('id', id);
 
+    holder.addEventListener('keydown', this.extractionBlock.bind(this, item));
```

Comment:
@Jenietoc @juliansg022 maybe using `extractBlock` make more sense, what do you think?

Reply:
Yes, `extractBlock ` is a better name for the function.